lib/Varien/Object.php

```diff
@@ -300,13 +300,15 @@ public function unsetOldData($key = null)
     }
 
     /**
-     * Retrieves data from the object
+     * Object data getter
      *
-     * If $key is empty will return all the data as an array
-     * Otherwise it will return value of the attribute specified by $key
+     * If $key is not defined will return all the data as an array.
+     * Otherwise it will return value of the element specified by $key.
+     * It is possible to use keys like a/b/c for access nested array data
      *
      * If $index is specified it will assume that attribute data is an array
-     * and retrieve corresponding member.
+     * and retrieve corresponding member. If data is the string - it will be explode
```

Comment:
If data is **a** string

Reply:
Mhh, im no native speaker, but "the" seem correct here.

Hhowever ... can you please use github suggestion function? (ctrl-g)

Reply:
The wording in that context is not exactly appropriate. It is not an error to turn this PR upside down, it can reformulated. As I can live with some of the OM translations, I can live with this wording.